Services/CryptionService.cs

```
@@ -0,0 +1,36 @@
+﻿using LethalCompany_Backend.Services.Interfaces;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace LethalCompany_Backend.Services;
+
+public class CryptionService : ICryptionService
+{
+    public string Decrypt(string key)
+    {
+        try
+        {
```

Try catching and returning exception message as password decryption? Please remove the try catch. in case of a crash im expecting a throw instead of a return